### ISSUE TITLE
Only bump tag with perf/feat/fix label

### ISSUE THREAD

**Issue by soumyamahunt** [TriPSs/conventional-changelog-action#152]
First of all I want to thank you for this awesome action, I was wondering if I can allow bump only for certain commit label like `perf`/`feat`/`fix`, while I would also like other commit labels to appear in my changelog like `doc`/`ci` etc. Right now from what I can see I have to hide the labels from changelog to be able to restrict bump for them. Is there some way I can achieve this?

**Comment by TriPSs** [comment]
Those things should be configureable with a different preset, default this action uses the `angular` preset.

**Comment by soumyamahunt** [comment]
@TriPSs thanks for the reply, I am using custom `config-file-path` which doesn't allow me for preset selection, although I have used the default value with only overriding required fields. Now when I run this action I would get a release even though there is no `perf`/`feat`/`fix` type commits:
> Recommended release type: patch
> Because: There are 0 BREAKING CHANGES and 0 features

Example of one [run](https://github.com/DartBuild/cocoapods-embed-flutter/runs/5520381059?check_suite_focus=true#step:7:1). Is there some way I can customize this behavior with config file? 

I followed the [spec docs](https://github.com/conventional-changelog/conventional-changelog-config-spec/blob/master/versions/2.1.0/README.md#types) but couldn't find anything that will help regarding this.
